chanlon27/chanlon27-Belfast.md

```diff
@@ -5,3 +5,6 @@
  - The Salty Dog :dog:
```

Comment:
The Salty Dog is a great restaurant in a town called Bangor in Northern Ireland.